hr_holidays_accrual/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018-2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Advanced Accrual Allocation',
+    'version': '12.0.1.0.0',
+    'category': 'Human Resources',
+    'website': 'https://github.com/OCA/hr',
+    'author':
+        'Brainbean Apps, '
+        'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'application': False,
+    'summary': 'Advanced accrual leaves allocation',
+    'depends': [
+        'hr_holidays',
+        'hr_employee_service',
+    ],
+    'external_dependencies': {
+        'python': [
+            'dateutil',
+        ],
+    },
+    'data': [
+        'security/hr_holidays_accrual_security.xml',
+        'security/ir.model.access.csv',
+        'wizard/hr_leave_allocation_accrual_calculator.xml',
```

Comment:
`wizard` --> `wizards` ? (I remember we talked this on another PR)

Reply:
but odoo uses `wizard`

Reply:
And oca wizards

Reply:
Yes, for consistency with other folders, better to use plural here, as seen in the template:

https://github.com/OCA/maintainer-tools/tree/master/template/module/wizards

or guidelines:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming

Reply:
Yep